### PR TITLE
[jsscripting] Rename library injection parameter & Improve docs

### PR DESCRIPTION
**Pull request by florian-h05**
Renames the `useIncludedLibrary` parameter to `injectionCachingEnabled` and updates label, description and options to be more descriptive of what the parameter is actually doing.

Also improve the docs to provide more guidance for add-on configuration.

### CODE REVIEW

File: bundles/org.openhab.automation.jsscripting/README.md [hunk 1]
```diff
@@ -39,13 +39,24 @@ to common openHAB functionality within rules including items, things, actions, l
...
+   - Yes: "Use Built-In Variables" (default)
+   - No: "Do Not Use Built-In Variables", which will allow you to decide what to import and really speed up script loading, but you need to manually import the library, which actually will slow down script loading again.
+2. Do I want to have a different version injected than the included one (`injectionCachingEnabled`)?
+   - Yes: "Do Not Cache Library Injection" and install your version to the `$OPENHAB_CONF/automation/js/node_modules` folder, which will slow down script loading, because the injection is not cached.
+   - No: "Cache Library Injection" (default), which will speed up the initial loading of a script because the library's injection is cached.
+
+Note that in case you disable caching or your code uses `require()` to import the library and there is no installation of the library found in the node_modules folder, the add-on will fallback to its included version.
 
-![openHAB Rule Configuration](doc/settings.png)
+In general, the first run of a script will take longer than the subsequent runs, because on the first run, the injection of both the globals (like `console`) and (if enabled) the library needs to be done.
```
Comment (jlaur) [inline bundles/org.openhab.automation.jsscripting/README.md:59]: Consider if this sentence could be broken into two separate sentences and/or simplified as it's rather complex to read. I had to read it a few times in parts without the parenthesis to grasp it ("on first run, the injection of (...) globals (...) and (...) the library needs to be done".

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks. I have provided only a few comments for the README. I know it's copied over from the openhab-js repository, so please advise how these comments should be handled (if you agree).

**Comment by florian-h05** [comment]
@jlaur I addressed your comments on the README. This part of the README is not copied from openhab-js, only a part of the README is shared.

**Review by jlaur — APPROVED** [review]
LGTM

**Pull request merged**